contributing.md

```diff
@@ -119,11 +119,19 @@ make docs
 
 ## Releasing
 
+Our current process for publishing a release of python-pachyderm consists of the following steps:
+
+- [ ] Ensure compatibility with the latest protos:
+  - [ ] Make sure `version.json` references the most recent compatible release of Pachyderm
+  - [ ] Run `make src/python_pachyderm/proto/v2` to pull the latest pachyderm code
```

Comment:
Maybe also a line after this saying to make sure you have the corresponding `pachctl` and `pachd` versions (so tests run in the correct environment)

Reply:
Good call, added.